test/datetypes.notz.test.js

```diff
@@ -0,0 +1,186 @@
+      query("SET @@session.time_zone = '"+localTZ+"';", function(err) {
+          assert.ok(!err);
+          done();
+      });
+  });
+
+  it('should create table', function(done) {
+    query("CREATE TABLE `DateModel` "+
+      "(`id` INT(11) NOT NULL AUTO_INCREMENT PRIMARY KEY,"+
+      "`datetimeField` datetime NULL,"+
+      "`timestampField` timestamp NULL,"+
+      "`timestampDefaultField` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,"+
+      "`dateField` date NOT NULL"+
+    ")", done);
+  });
+
+  it('should create a model instance with dates with local TZ', function(done) {
+    DateModel.create({
+        datetimeField: date,
+        timestampField: date,
+        dateField: dateOnly
+    }, function(err, obj) {
+      assert.ok(!err);
+      done();
+    });
+  });
+  it('should get model instance', function(done) {
+      DateModel.findOne({where: {id: 1}}, function(err, found) {
+        assert.ok(!err);
+        assert.equal(found.datetimeField.toISOString(), date.toISOString());
+        assert.equal(found.timestampField.toISOString(), date.toISOString());
+        assert.equal(found.dateField.toISOString(), dateOnly.toISOString());
+        done();
+      });
+  });
+  it('timestampField shoud equal DEFAULT CURRENT_TIMESTAMP field', function(done) {
+      DateModel.findOne({where: {id: 1}}, function(err, found) {
+        assert.ok(!err);
+        assert.equal(found.timestampField.toISOString(), found.timestampDefaultField.toISOString());
+        done();
+      });
+  });
+  it('should find model instance by datetime field', function(done) {
+      DateModel.findOne({where: {datetimeField: date}}, function(err, found) {
+        assert.ok(!err);
+        assert.equal(found.datetimeField.toISOString(), date.toISOString());
+        assert.equal(found.timestampField.toISOString(), date.toISOString());
+        assert.equal(found.dateField.toISOString(), dateOnly.toISOString());
+        done();
+      });
+  });
+  it('should find model instance by timestamp field', function(done) {
+      DateModel.findOne({where: {timestampField: date}}, function(err, found) {
+        assert.ok(!err);
+        assert.equal(found.datetimeField.toISOString(), date.toISOString());
+        assert.equal(found.timestampField.toISOString(), date.toISOString());
+        assert.equal(found.dateField.toISOString(), dateOnly.toISOString());
+        done();
+      });
+  });
+  it('should find model instance by date field', function(done) {
+      DateModel.findOne({where: {dateField: dateOnly}}, function(err, found) {
+        assert.ok(!err);
+        assert.equal(found.datetimeField.toISOString(), date.toISOString());
+        assert.equal(found.timestampField.toISOString(), date.toISOString());
+        assert.equal(found.dateField.toISOString(), dateOnly.toISOString());
+        done();
+      });
+  });
+
+  it('should disconnect when done', function(done) {
+    db.disconnect();
+    done();
+  });
+});
+
+function setup(done) {
```

Comment:
Don't need any of these setup AFAIK. All the models/tables should be on the default database the test runs against.